base-notebook/Dockerfile

```diff
@@ -40,7 +57,10 @@ RUN apt-get update --yes && \
     #   processes and such of the actual executable we want to start, see
     #   https://github.com/krallin/tini#why-tini for details.
     tini \
-    wget && \
+    wget \
+    curl \
```

Comment:
```suggestion
```

Reply:
Ya, this is a bit of a rough draft. I'd like to squash, assuming I get it running. If you approve, of course. I don't want to force push too much. :wink:

Reply:
It's ok if you don't squash. I don't like push forces, especially when someone already reviewed the content.
Just keep it the way it is, it's ok for us to have "Fix this" and "Fix that" comments.